Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.9-slim
```

Comment:
This file looks okay. Most of our projects have put these in a subdirectory like "dockerfiles/Dockerfile" to clean up the root a little.

Reply:
Done